core/store/src/trie/state_parts.rs

```diff
@@ -71,7 +72,7 @@ impl Trie {
         if part_id == num_parts {
             return Ok(vec![16]);
         }
-        let root_node = self.retrieve_node(&self.root)?.1;
+        let (_bytes, root_node) = self.retrieve_node(&self.root)?;
```

Comment:
Either works, but why change? It just adds noise to the commit.  Same in other places in this file.